Add a note that SharedArrayBuffer Origin Trial is only available through http header (#679)
GoogleChrome/developer.chrome.com: site/en/blog/enabling-shared-array-buffer/index.md

Chrome.

1. [Request a token]({{origin_trial.url}}) for your origin.
2. Add the token to your pages using an `Origin-Trial` HTTP header. The
   resulting response header should look something like: `Origin-Trial:
   TOKEN_GOES_HERE`

{% Aside 'caution' %}

> You can only enable the origin trial using an `Origin-Trial` HTTP header. Please
> do not use a meta tag.

Reviewer: ```suggestion
You can only enable the origin trial using an `Origin-Trial` HTTP header.
Do not use a meta tag for this purpose.
```

Reviewer: Just a note that it's best practice to be direct and avoid using "please" for instructions, it's not impolite :)

> Do not use a meta tag for this purpose.

Reviewer 2: ```suggestion
Do not use a meta tag.
```

{% endAside %}

> You can verify that it's working properly by installing and testing with [Chrome
> 91 beta](https://www.google.com/chrome/beta/).

Reviewer: ```suggestion
To verify that it's working properly, install [Chrome
91 beta](https://www.google.com/chrome/beta/) for testing.
```

Reviewer: Also best practice to avoid passive voice.
